requirements/core-requirements.yaml

```diff
@@ -81,3 +81,7 @@ Jinja2-windows:
 matplotlib:
   pip_release: matplotlib
   max_major_version: 3
+
+psutil:
+  pip_release: psutil
+  max_major_version: 5
```

Comment:
We also have `psutil` in `requirements/gateway-requirements.yaml`. Let's remove it there to avoid duplicate dependecies.

Reply:
good call, done